modules/apps/main.go

```diff
@@ -24,9 +24,11 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
 	"os"
 	"strings"
 
+	"github.com/Nanocloud/community/modules/apps/lib/apps"
```

Comment:
iaas-module doesn't get a name for its import too, I guess that's OK for now but we'll have to change that soon